core/src/test/scala/io/finch/MethodSpec.scala

```diff
@@ -3,6 +3,8 @@ package io.finch
 import cats.effect.IO
 import com.twitter.finagle.http.Response
 import com.twitter.util.{Future => TwitterFuture}
+import io.finch.instances.scalaFuture._
```

Comment:
also moved scala and twitter future into `instance` package because they are just implementation of typeclass `?~>Effect`, as cats convention
it still works if leave it in Mapper but i think it shouldn't belong to mapper anymore, mapper is generic for any kind now

Reply:
Well, that's exactly my point. We could avoid imports here and there using home-backed type class with companion object.
https://github.com/finagle/finch/blob/b03b88ad2339cebc1a106bb4f203d64895fde56e/core/src/main/scala/io/finch/internal/LiftToEffect.scala

Reply:
Still not getting your point. You can move where ever you want, they are implicits
I just think they should organized this way in cats philosophy
Imagine If I implement your type class LiftToEffect for my type in my code, those future will be auto import to my context which I don't need at all.
How is that better than explicitly import

Reply:
> I just think they should organized this way in cats philosophy

Based on my experience working with different people through the years, this is one of the most frustrating pain points they had with Scala. It's not enough to write an actual code anymore, but also requires a user to know what and when to import.

> those future will be auto import to my context which I don't need at all.

They aren't auto-imported, but only by need. As far as you don't use these specific types in your context, compiler doesn't care much about the those implicit instances. This pattern is widely adopted in finch. Examples:
- [Encode](https://github.com/finagle/finch/blob/master/core/src/main/scala/io/finch/Encode.scala#L57) and [DecodeEntity](https://github.com/finagle/finch/blob/6f5006fd4395c8c8066692c44c889bdd0ea43d5d/core/src/main/scala/io/finch/DecodeEntity.scala#L28)
- [ToResponse](https://github.com/finagle/finch/blob/6f5006fd4395c8c8066692c44c889bdd0ea43d5d/core/src/main/scala/io/finch/ToResponse.scala#L72)

Reply:
> It's not enough to write an actual code anymore but also requires a user to know what and when to import

I totally understand what your concern is and I think
cats already have pretty good pattern to cater both kind of users

for peeps don't know what to import, they simply import everything and let the compile pick for them
```scala
import cats.implicits._
```
for peeps know exactly what they need, i.e. functor instance of list
```scala
import cats.instances.list._
```
they both behave the same from user point of view but I believe the second one is more gentle to the compiler and more explicit about the intention

> They aren't auto-imported, but only by need

they aren't at run time but are there for compiler search

> This pattern is widely adopted in finch

i think decoupling typeclass and data type is the pattern that functional programming different from OO. it make sense to put all instance of data type in data type's companion object but not in typeclass's companion object 
i.e.
when` import cats.effect.IO` you have all instances of datatype IO for typeclass Monad, Effect etc

but when you `import cats.effect.Effect` you should only have typeclass `Effect`, because when i import the typeclass, it means I want to implement it for my data type, which means I'm using custom data type, not IO, all instance of IO for that typeclass is the last thing I need.

since finch adopted cats, i reckon keeping only one pattern is easier for both maintainer and user

Reply:
I see your point here, but in my opinion it's rather subjective and based on the experience with certain environment. My argument comes from the UX point of view, rather than FP (or any other) purity. Extrapolating the example of `Encode` and `ToResponse` toward the state where every implicit is placed under a separate package, we end up either with dozens of imports for each primitive type like `Int`, `Boolean` or `String`, or introduce the single import of truth `io.finch.implicits._`, which is, sorry for being blunt, but just a cargo cult.

Reply:
From UX point of view, I reckon user choose Finch over other is because fp since Finch branding itself so isn't it. At least I am because it's compatible with cats ecosystem and I expected seamless UX as cats.
If user come from other experience other than cats isn't finatra the a better choice then

Reply:
I'd like to respond as user, not a maintainer. The team I work with chose Finch 3 years ago because it was easy to use, very concurrent, and gave us all the benefits of Finagle to use as we needed. We've never followed how Cats does usability things (and my teammates have struggled with Cats' conventions), I that find while I understand the import strategy, I'm constantly teaching it or reminding others of it. While I'd personally be willing to accept another import (and this PR), we can't discount the users. I'm with @sergeykolbasov on this one.I think we should let it be discovered by the compiler.